psi4/driver/p4util/fcidump.py

```diff
@@ -36,23 +36,29 @@
 from psi4.driver.procrouting.proc_util import check_iwl_file_from_scf_type
 
 from psi4 import core
-from .exceptions import ValidationError, TestComparisonError
 
 
-def fcidump(wfn, fname='INTDUMP', oe_ints=None):
+def fcidump(wfn, fname='INTDUMP', oe_ints=None, write_pntgrp=False):
     """Save integrals to file in FCIDUMP format as defined in Comp. Phys. Commun. 54 75 (1989)
     Additional one-electron integrals, including orbital energies, can also be saved.
     This latter format can be used with the HANDE QMC code but is not standard.
+    This function converts the irrep labels from Cotton's order (used in psi4)
+    to molpro ordering. Since the latter is ambiguous unless the point group is
+    specificied, this function has an option to write the point group label in the
+    FCIDUMP file. This, however, is not part of the standard.
 
     :returns: None
 
-    :raises: ValidationError when SCF wavefunction is not RHF
+    :raises: ValidationError when SCF wavefunction is not RHF, ROHF, or UHF
```

Comment:
I am not sure the original format specification allows ROHF.

Reply:
I am not sure either. So in this case I avoid saving the eigenvalues. But the 1-/2-e integrals are still well defined. It might be difficult to reconstruct a ROHF wave function without specifying the orbital energies, but still it can be used to exchange integrals. The more I use FCIDUMP the more it is clear that this format is not designed to be general enough.